src/node-sass/test/fixtures/compressed/index.css

```diff
@@ -0,0 +1,11 @@
+#navbar {
+  width: 80%;
+  height: 23px; }
+
+#navbar ul {
+  list-style-type: none; }
+
+#navbar li {
+  float: left; }
+  #navbar li a {
```

Comment:
For css/scss variables kindly use BEM guidelines, have a look at this [tuitorial](https://assist-software.net/blog/css-guideline-tutorial-bem-sass) and [example](https://github.com/jalyna/bem-scss-styleguide).

Reply:
@justmesam these files were auto-created by node-sass. That was the mistake i have removed the files because we don't need them